k8s/helmfile/env/local/kube-prometheus-stack.values.yaml.gotmpl

```diff
@@ -22,7 +22,7 @@ prometheus:
     image:
       registry: quay.io
       repository: prometheus/prometheus
-      tag: v2.42.0
+      tag: v2.49.1
```

Comment:
Not entirely sure what you mean by "this"? If I understand it correctly, production and staging run an entirely different image, so we'll need to override this again in its entirety in local.

Reply:
I was hoping that there would be some way I could imagine overriding the local image value to be the one from the chart rather than having to give an explicit value. This would mean that next time we update the chart we don't have to remember to also update this